hawc/apps/summary/models.py

```diff
@@ -149,6 +153,65 @@ def print_node(node, depth):
                 print_node(node, 2)
 
 
+class SummaryTable(models.Model):
+    objects = managers.SummaryTableManager()
+
+    GENERIC = 0
+    EVIDENCE_PROFILE = 1
+    EVIDENCE_INTEGRATION = 2
+
+    TABLE_CHOICES = (
+        (GENERIC, "Generic table"),
+        (EVIDENCE_PROFILE, "Evidence profile table"),
+        (EVIDENCE_INTEGRATION, "Evidence integration table"),
+    )
+
+    assessment = models.ForeignKey(Assessment, on_delete=models.CASCADE)
+    title = models.CharField(max_length=128)
+    slug = models.SlugField(
+        verbose_name="URL Name",
+        help_text="The URL (web address) used to describe this object "
+        "(no spaces or special-characters).",
+    )
+    content = models.JSONField(default=dict)
+    table_type = models.PositiveSmallIntegerField(choices=TABLE_CHOICES, default=GENERIC)
+    published = models.BooleanField(
+        default=False,
+        verbose_name="Publish table for public viewing",
+        help_text="For assessments marked for public viewing, mark table to be viewable by public",
+    )
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+    def get_assessment(self):
+        return self.assessment
+
+    def get_content_schema_class(self):
+        if self.table_type == self.GENERIC:
+            return GenericTable
+
+    def get_table(self):
+        return self.get_content_schema_class().parse_obj(self.content)
+
+    def to_report(self):
+        table = self.get_table()
+        return ReportExport(docx=table.to_docx(), filename=self.slug)
+
+    def clean(self):
```

Comment:
Since the `clean` method just makes sure the content is parsable by our pydantic models, I added some parse tests to `test_generic.py` at your recommendation. As for this particular example I think its valid; I just take it as a 2x2 table with empty cells.